rdt/__init__.py

```diff
@@ -68,6 +69,8 @@ def get_demo(num_rows=5):
     dollars_spent = np.around(np.random.uniform(0, 100, size=num_rows), decimals=2)
     dollars_spent[np.random.random(size=num_rows) > 0.8] = np.nan
 
+    np.random.set_state(random_state)
```

Comment:
You could put this in a 
```
try:
     <code>
finally:
     np.random.set_state(random_state)
```
block to be extra sure that the random state is set to the original value.